### ISSUE TITLE
Add icon for chess tables

### ISSUE THREAD

**Issue by Tomasz-W** [gravitystorm/openstreetmap-carto#3039]
Currently, there are no difference between simple picnic tables (tagged as leisure=picninc_table) and chess tables (tagged as leisure=picnic_table + sport=chess). 
https://wiki.openstreetmap.org/wiki/Tag:sport=chess

It may be confusing for map users.

Icon project:

![sport chess2](https://user-images.githubusercontent.com/25656654/35471265-fb6275a6-0357-11e8-8c07-3d1495c84291.png)

14x14:  ![sport chess](https://user-images.githubusercontent.com/25656654/35471266-02fca566-0358-11e8-9af5-035e12fe11f0.png)

**Comment by polarbearing** [comment]
We currently render no sports icon at all. If we would start doing this, a chess table would be much rarer than other kinds of sport. 
I'd consider leisure=picnic_table wrong tagging for a chess table, a table tennis table, etc.

**Comment by Tomasz-W** [comment]
Related to #844

**Comment by Tomasz-W** [comment]
> I'd consider leisure=picnic_table wrong tagging for a chess table, a table tennis table, etc.

Due to OSM Wiki, correct tagging of tennis tables is leisure=pitch + sport=table_tennis
https://wiki.openstreetmap.org/wiki/Tag:sport%3Dtable_tennis

**Comment by dieterdreist** [comment]
sent from a phone

> On 28. Jan 2018, at 00:06, polarbearing <notifications@github.com> wrote:
> 
> I'd consider leisure=picnic_table wrong tagging for a chess table, a table tennis table, etc.


for me leisure=picnic_table sport=chess is ok, and I wouldn’t see this primarily as a sports feature.

**Comment by kocio-pl** [comment]
I like this icon and it wouldn't be the first sport icon we use - we show green icon for golf, mini golf, water park and fitness station. Who wants to prepare the code?

**Comment by matthijsmelissen** [comment]
I think chess tables are too specific for our purposes.

**Comment by dieterdreist** [comment]
sent from a phone

> On 1. Feb 2018, at 23:33, Matthijs Melissen <notifications@github.com> wrote:
> 
> I think chess tables are too specific for our purposes.


I’ve had a look and globally there are 264 of such objects, which is a really small number. On the other hand the icon is very similar to picnic table, and we do render those, so it is not introducing a new feature but refining certain cases of an existing one with a little accent

**Comment by Tomasz-W** [comment]
I've prepared this icon because OSM styles should show rather certain elements functions than just it's shapes. Chess tables are for playing chess, not e.g. eating sandwiches (of course you can do it there, but it's not a main function). Anyway, I still would like to see this two types of tables look different.

**Comment by polarbearing** [comment]
Yes, but we already recognised that tagging chess tables as ```leisure=picnic_table``` is wrong already. They should follow the table_tennis example and tag themselves as ```leisure=pitch```, where you can have an extra tag to distinguish ground chess fields from tables. .

I wonder if leisure=pitch is rendered on a node?

**Comment by kocio-pl** [comment]
Hm, I also don't think it has anything to do with picnic, it's just abusing the table tag, so new tagging is needed. I will close this ticket now, but it can be reopened once the new tagging is established (let's talk about it on Tagging list).

@Tomasz-W Please post this icon as a gist, so it can be easily reached when needed.

**Comment by Tomasz-W** [comment]
https://gist.github.com/Tomasz-W/597055a625898c88eb30eb3eb398628a

**Comment by dieterdreist** [comment]
2018-02-02 12:01 GMT+01:00 kocio-pl <notifications@github.com>:

> Hm, I also don't think it has anything to do with picnic, it's just
> abusing the table tag, so new tagging is needed. I will close this ticket
> now, but it can be reopened once the new tagging is established (let's talk
> about it on Tagging list).
>

all cases of open air public chess tables that I have seen were just normal
tables with a checker board on them. Most of the time people were not
playing chess but using the table otherwise. leisure=pitch for a table
tennis table is tagging for the renderer, it is not a pitch (FYI, the
meaning of "pitch" is sports field).

**Comment by kocio-pl** [comment]
Well, you're right, it's complicated case... Is anybody ready to prepare the code?

**Comment by polarbearing** [comment]
The wiki definition for pitch is "area designed for playing a particular sport" and thus a bit wider than the natural language. You would need to discuss that tagging as it is wide-spread.

Besides the tables (which I have seen in two forms, with a dedicated table-top with [the squares laid in](https://commons.wikimedia.org/wiki/File:Chess_table_escacs.JPG), or as [regular tables to bring your own board](https://commons.wikimedia.org/wiki/File:Chess_board_in_park,_kiev.JPG)), there are [outdoor chess fields on the ground](https://commons.wikimedia.org/wiki/File:Innsbruck_17.jpg).

**Comment by dieterdreist** [comment]
2018-02-04 11:39 GMT+01:00 polarbearing <notifications@github.com>:

> The wiki definition for pitch is "area designed for playing a particular
> sport" and thus a bit wider than the natural language. You would need to
> discuss that tagging as it is wide-spread.
>
> Besides the tables (which I have seen in two forms, with a dedicated
> table-top with the squares laid in
> <https://commons.wikimedia.org/wiki/File:Chess_table_escacs.JPG>, or as regular
> tables to bring your own board
> <https://commons.wikimedia.org/wiki/File:Chess_board_in_park,_kiev.JPG>),
> there are outdoor chess fields on the ground
> <https://commons.wikimedia.org/wiki/File:Innsbruck_17.jpg>.
>

yes, there are these outdoor chess fields which would better qualify as
pitch than a table, although it seems a bit of a stretch as well

**Comment by Tomasz-W** [comment]
Just for the note. current tagging scheme is:
* for chess tables: `leisure=picnic_table` + `sport=chess`
* for ground chess fields: `leisure=pitch` + `sport=chess`
